client/src/pages/Home/Register.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import {
+  ButtonContainer,
+  LeftCutout,
+  RightCutout,
+} from '../../components/Button/styled'
+import { PromptLink } from '../../components/AccountPrompt/styled'
+import { SignUpLink, SignUpContainer, Space } from './styled'
+
+const Register = () => {
+  return (
+    <SignUpContainer id="signUp">
+      <Space />
+      <ButtonContainer>
+        <LeftCutout />
+        <p>
+          <SignUpLink to="/signup">Sign Up</SignUpLink>
+        </p>
+        <RightCutout />
+      </ButtonContainer>
+      <PromptLink to="/login">i have an account</PromptLink>
```

Comment:
I know this font automatically capitalizes everything, but we should still use proper casing